### PR TITLE
IDEMPIERE-5362 - Attribute Dialog UX improvments 2022 aug

### PR DESCRIPTION
**Pull request by PeterTakacs300**
JIRA ticket: https://idempiere.atlassian.net/browse/IDEMPIERE-5362

### CODE REVIEW

**Comment by PeterTakacs300** [comment]
Before:
Screenshot 2022-08-02 at 10 59 17

After:
Screenshot 2022-08-02 at 10 54 58

Problem:
Screenshot 2022-08-02 at 10 55 51

I temporarily hardcoded the height to 500px, I would need help with setting max-height to the dialog, so there would not be the empty space seen on the picture above, when we have less attributes.

**Comment by hengsin** [comment]
hi @PeterTakacs300 , you have to use css or setstyle to set min or max height (https://www.w3schools.com/cssref/pr_dim_min-height.asp)

**Comment by PeterTakacs300** [comment]
Current state

Max height:
Screenshot 2022-08-03 at 08 59 49

Shrinking when there are less attributes:
Screenshot 2022-08-03 at 09 00 09

File: org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/window/WPAttributeDialog.java [hunk 1]
```diff
@@ -215,13 +218,22 @@ private void init () throws Exception
 		mainLayout.setParent(this);
 		ZKUpdateUtil.setHflex(mainLayout, "1");
 		ZKUpdateUtil.setVflex(mainLayout, "min");
+		mainLayout.setStyle("max-height: 600px;");
```
Comment (hengsin) [inline org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/window/WPAttributeDialog.java:221]: This wouldn't works well on mobile with smaller height. one option is to do it with condition, i.e `if (ClientInfo.maxHeight(600) mainLayout.setStyle("max-height: 100%;"); else mainLayout.setStyle("max-height: 600px;");`

As a safety net, we can setMaxizable(true) for the dialog. This give user a way to make the dialog usable if there's problem with default sizing set by the application.

**Comment by PeterTakacs300** [comment]
https://user-images.githubusercontent.com/93127072/182789155-f007b4b9-f3d2-4dde-bbaf-0dc38f34b73b.mp4

**Review by hengsin — CHANGES_REQUESTED** [review]
please update ticket with the details of other changes.

**Comment by norbertbede** [comment]
done

**Review by CarlosRuiz-globalqss — CHANGES_REQUESTED** [review]
Please check comments in ticket

**Review by CarlosRuiz-globalqss — APPROVED** [review]

**Comment by CarlosRuiz-globalqss** [comment]
Tested, I find this OK.  @hengsin  ?

**Comment by hengsin** [comment]
ok for me.

**Review by hengsin — APPROVED** [review]
tested

**Pull request merged**